e-nable/LimbForge-Web-App
# 105

closes https://github.com/limbforge/Limbforge/issues/105


## Description:
This PR adds access request for Limbforge users. 


## QA:
- sign up with a new account on the [staging env](https://limbforge-staging1.herokuapp.com/) with an email you have access to.
- [x]  You should see a "we are in version beta screen" and be able to click "request access"
- [ ] After clicking "request access" you should be forwarded to a "thanks" screen
- log into the learnmore@limbforge.org email account
- [x] You should see an email with the request for access + instructions on how to give them access.
Make sure you are signed into heroku in your console:
run
```
heroku login
```
*and enter all of your credentials for the (learnmore@limbforge.org) account
Additionally, Be sure to use the correct app name:
replace this line in the commands:
```
heroku run rails console --app limbforge
```
with
```
heroku run rails console --app limbforge-staging1
```
- [ ] After running through the instructions on the email ^^ you should see a confirmation email in your email that you signed up to Limbforge with.
- [ ] You should be able to sign into the staging app for limbforge now.



## Notes:
Once this is pushed to production we need a list of emails that we want to automatically give access to. @andreasbastian can you list them in this PR?

*Additionally, once this PR is pushed to any new site be sure to migrate the database!

@WomenWhoCode/website-reviewers
